src/app/open-source/page.tsx

```diff
@@ -26,6 +27,16 @@ export default function OpenSourcePage() {
                             more exciting projects planned for the future!
                         </p>
                     </section>
+                    <section className="mb-8">
+                        <Button
+                            type="button"
+                            colour="orange"
+                            width="w-full"
+                            href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
+                        >
+                            Join Us Now
+                        </Button>
+                    </section>
```

Comment:
Can you make the button width smaller on desktop so it doesn't span the whole container? Also make the text grey instead of white for better contrast.

Reply:
sure, does 1/2 width look good to you?